pkg/engine/backend/backend.go

```diff
@@ -0,0 +1,119 @@
+package backend
+
+import (
+	"fmt"
+	"path/filepath"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/zclconf/go-cty/cty"
+	"github.com/zclconf/go-cty/cty/gocty"
+	backendInit "kusionstack.io/kusion/pkg/engine/backend/init"
+	"kusionstack.io/kusion/pkg/engine/states"
+	"kusionstack.io/kusion/pkg/engine/states/local"
+	"kusionstack.io/kusion/pkg/util/i18n"
+)
+
+// backend config state storage type
+type Backend struct {
```

Comment:
package name is  backend, I prefer to rename struct to `Storage`, and rename `StorageType` to `Type`